event_project/models/project_task.py

```diff
@@ -1,33 +1,17 @@
-# -*- encoding: utf-8 -*-
-##############################################################################
-#
-#    OpenERP, Open Source Management Solution
-#    This module copyright :
-#        (c) 2014 Antiun Ingenieria, SL (Madrid, Spain, http://www.antiun.com)
-#                 Endika Iglesias <endikaig@antiun.com>
-#                 Antonio Espinosa <antonioea@antiun.com>
-#                 Javier Iniesta <javieria@antiun.com>
-#
-#    This program is free software: you can redistribute it and/or modify
-#    it under the terms of the GNU Affero General Public License as
-#    published by the Free Software Foundation, either version 3 of the
-#    License, or (at your option) any later version.
-#
-#    This program is distributed in the hope that it will be useful,
-#    but WITHOUT ANY WARRANTY; without even the implied warranty of
-#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
-#    GNU Affero General Public License for more details.
-#
-#    You should have received a copy of the GNU Affero General Public License
-#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
-#
-##############################################################################
+# -*- coding: utf-8 -*-
+# See README.rst file on addon root folder for license details
 
-from openerp import models, fields
+from openerp import models, fields, api
 
 
 class ProjectTask(models.Model):
     _inherit = 'project.task'
 
-    anticipation_days = fields.Integer(
-        string='Event anticipation days', help='Event anticipation days')
+    event_id = fields.Many2one(
+        comodel_name='event.event', compute='_compute_event', store=True,
+        string="Related event", readonly=True)
```

Comment:
You can simplify this by make this instead a `Related`field for `project_id.event_id`